### PR TITLE
feat: mobile page

### LINKED ISSUE #105: Mobile view
We'll need to show something when the user accesses the editor view for a project with a small/not supported resolution

### PR DESCRIPTION
**Pull request by belohlavek**
kapture 2019-02-22 at 11 17 38

On submit the user is identified, tracked and redirected to the contest page.
Mobile route is not accesible on desktop (you get redirected to the homepage).

Closes #105

### CODE REVIEW

File: src/components/MobilePage/MobilePage.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import { Header, Field, Button, Form } from 'decentraland-ui'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { getAnalytics } from 'decentraland-dapps/dist/modules/analytics/utils'
+
+import { Props, State } from './MobilePage.types'
+
+import './MobilePage.css'
+import { isMobile } from 'decentraland-dapps/dist/lib/utils'
```
Comment (cazala) [inline src/components/MobilePage/MobilePage.tsx]: this import should go up 🙃

File: src/components/MobilePage/MobilePage.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    return (
+      <div className="MobilePage">
+        <Header size="large">{t('mobilepage.title')}</Header>
+        <p className="subtitle">{t('mobilepage.subtitle')}</p>
+        <Form onSubmit={this.handleSubmit}>
+          <p className="message">{t('mobilepage.message')}</p>
+          <div className="form-container">
+            <Field type="email" icon="asterisk" placeholder="mail@domain.com" value={email} onChange={this.handleEmailChange} required />
+            <Button primary size="medium">
+              {t('global.send').toUpperCase()}
```
Comment (cazala) [inline src/components/MobilePage/MobilePage.tsx]: this can be done with css: `text-transform: uppercase`

Comment (belohlavek) [reply src/components/MobilePage/MobilePage.tsx]: yup, will need to change it on the contest too I think (copy pasted)

Comment (cazala) [reply src/components/MobilePage/MobilePage.tsx]: that should be a style of the `<Button>` component tho

Comment (cazala) [reply src/components/MobilePage/MobilePage.tsx]: I just checked `decentraland-ui`, it already does that, so you can just remove the `.toUpperCase()` and it should still show all in caps

**Comment by belohlavek** [comment]
waiting for nico's feedback on the routing

**Pull request merged**